woocommerce/woocommerce-ios
# [Mobile Payments] Prompt user to update the specific out of date plugin

Closes: #5529

### Description
- Adds image (PDF) for Stripe plugin
- Renames CardPresentPaymentsOnboardingUseCase Constants to CardPresentPaymentsPlugins so we can use them in related onboarding views and changes it to use computed properties
- Adds associated value to onboarding case pluginUnsupportedVersion
- Changes InPersonPaymentsPluginNotSupportedVersionView.swift to accept the associated value so we have access to the correct title to display and so we can calculate the image to display

### Nuances
- In this particular PR, InPersonPaymentsPluginNotSupportedVersionView has image as a calculated property. I'm seriously considering what to do for the other related views to avoid repeating myself. Suggestions welcome. Edit: I know what I want to do - I'll make a new child view that the plugin enum is passed to that renders the appropriate image. I'll update the PR. We can use that new child view in the views that need it.

### Testing instructions
- On a site with the Stripe Gateway installed, edit the plugin header to indicate plugin version 5.8.0 or older
- Select that site and then Menu > gear (Settings) > In-Person Payments
- Ensure you are prompted to update the WooCommerce Stripe Gateway (including the picture being correct)
- Put the version back the way it was
- Repeat with a site with the WCPay. In that case, edit the plugin header to indicate plugin version 3.2.0 or older

### Screenshots

<img src="https://user-images.githubusercontent.com/1595739/150230070-24743f04-4aec-4466-9e44-ad64e5e32eea.PNG" width=70%/>

---
